### ISSUE TITLE
Error on website, no map or tools available

### ISSUE THREAD

**Issue by althio** [openstreetmap/openstreetmap-website#2311]
![www openstreetmap org_history](https://user-images.githubusercontent.com/9863567/61204670-241fc580-a6ee-11e9-8455-ebecc4c808e1.png)

Console:
```
application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2 

jQuery.Deferred exception: querystring is not defined ReferenceError: querystring is not defined
    at Object.parseHash (https://www.openstreetmap.org/assets/application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:4:16170)
    at Object.mapParams (https://www.openstreetmap.org/assets/application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:4:15161)
    at HTMLDocument.<anonymous> (https://www.openstreetmap.org/assets/index-110cbde4026ad38824a347370cd7688971c396c9e129f9b860ef7d5ae476f1ff.js:1:6904)
    at e (https://www.openstreetmap.org/assets/application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2:9165)
    at o (https://www.openstreetmap.org/assets/application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2:9471) undefined
Tt.Deferred.exceptionHook @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
o @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
setTimeout (async)
(anonymous) @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
u @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
fireWith @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
fire @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
u @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
fireWith @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
ready @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
i @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:1
application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:4 

Uncaught ReferenceError: querystring is not defined
    at Object.parseHash (application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:4)
    at Object.mapParams (application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:4)
    at HTMLDocument.<anonymous> (index-110cbde4026ad38824a347370cd7688971c396c9e129f9b860ef7d5ae476f1ff.js:1)
    at e (application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2)
    at o (application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2)
parseHash @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:4
mapParams @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:4
(anonymous) @ index-110cbde4026ad38824a347370cd7688971c396c9e129f9b860ef7d5ae476f1ff.js:1
e @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
o @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
setTimeout (async)
Tt.readyException @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
(anonymous) @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
e @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
o @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
setTimeout (async)
(anonymous) @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
u @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
fireWith @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
fire @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
u @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
fireWith @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
o @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
setTimeout (async)
(anonymous) @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
u @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
fireWith @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
fire @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
u @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
fireWith @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
ready @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:2
i @ application-fdc4540785853975539f6279269d1bccd930dfeb0a52d93ee79f184a531588ca.js:1```

**Comment by strump** [comment]
Looks like this issue was caused by commit [#8077500](https://github.com/openstreetmap/openstreetmap-website/commit/807750023aa02896117a1d5d67a750eccd31d90b) "Fix eslint warning".

**Comment by tomhughes** [comment]
Fixed in a47c0a4411825f48e447641094678b15df3b2f74.
